exercises/practice/spiral-matrix/tests/spiral-matrix.rs

```diff
@@ -9,46 +9,42 @@ fn empty_spiral() {
 #[test]
 #[ignore]
 fn size_one_spiral() {
-    let mut expected: Vec<Vec<u32>> = Vec::new();
-    expected.push(vec![1]);
+    let expected: Vec<Vec<u32>> = vec![vec![1]];
     assert_eq!(spiral_matrix(1), expected);
 }
 #[test]
 #[ignore]
 fn size_two_spiral() {
-    let mut expected: Vec<Vec<u32>> = Vec::new();
-    expected.push(vec![1, 2]);
-    expected.push(vec![4, 3]);
+    let expected: Vec<Vec<u32>> = vec![vec![1, 2], vec![4, 3]];
     assert_eq!(spiral_matrix(2), expected);
 }
 
 #[test]
 #[ignore]
 fn size_three_spiral() {
-    let mut expected: Vec<Vec<u32>> = Vec::new();
-    expected.push(vec![1, 2, 3]);
-    expected.push(vec![8, 9, 4]);
-    expected.push(vec![7, 6, 5]);
+    let expected: Vec<Vec<u32>> = vec![vec![1, 2, 3], vec![8, 9, 4], vec![7, 6, 5]];
```

Comment:
```suggestion
    #[rustfmt::skip]
    let expected: Vec<Vec<u32>> = vec![
        vec![1, 2, 3], 
        vec![8, 9, 4], 
        vec![7, 6, 5],
    ];
```

This is the kind of place where I think manual formatting is appropriate: it helps users visualize the spiral in a way that rustfmt doesn't realize would be helpful to preserve. Not essential, but nice.

Reply:
Oh, cool! TIL `#[rustfmt::skip]`. 
I totally agree. The grid representation seems more readable to me.